corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -1242,6 +1243,7 @@ def fetch_case_transaction_forms(case, transactions, updated_xforms=None):
 
         updated_xform_ids = set(updated_xforms_map)
         form_ids_to_fetch = list(form_ids - updated_xform_ids)
+        form_load_counter("fetch_case_transaction_forms", case.domain)(len(form_ids_to_fetch))
```

Comment:
should this not be tagged as `case_rebuild` since that's why it's fetching the transactions?

Reply:
This is also called from `SqlCaseUpdateStrategy.reconcile_transactions()` which is called during form processing. I wasn't sure if they both could be classified as "case rebuild", but I guess the answer is yes. I'll change it.

Reply:
2fc51193b358c3a803d227da2829dd116eb88b44